server/views/pages/basic-details.njk

```diff
@@ -0,0 +1,238 @@
+
+  {% from "moj/components/side-navigation/macro.njk" import mojSideNavigation %}
+  {% from "govuk/components/phase-banner/macro.njk" import govukPhaseBanner %}
+  {% from "govuk/components/input/macro.njk" import govukInput %}
+  {% from "govuk/components/select/macro.njk" import govukSelect %}
+  {% from "moj/components/button-menu/macro.njk" import mojButtonMenu %}
+  {% from "moj/components/date-picker/macro.njk" import mojDatePicker %}
+  {% from "govuk/components/radios/macro.njk" import govukRadios %}
+
+<form method="post">
+
+  {% set errorList =[
+    { text: errorMessages.dateOfLetter.text, href: "#date-of-letter" } if errorMessages.dateOfLetter else None,
+    { text: errorMessages.officeReferenceNumber.text, href: "#office-reference" } if errorMessages.officeReferenceNumber else None,
+    { text: errorMessages.pdfRenderError.text } if errorMessages.pdfRenderError else None
+  ] | reject("undefined") %}
+
+  {% if errorMessages | length > 0 %}
+    {% from "govuk/components/error-summary/macro.njk" import govukErrorSummary %}
+
+    {{ govukErrorSummary({ titleText: "There is a problem", errorList: errorList }) }}
+  {% endif %}
+
+
+  <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-one-third">
+      {% set alternateAddressHtml %}
+        {{ govukSelect({
+          id: "alternate-address",
+          name: "alternateAddress",
+          label: {
+            text: "Please pick an alternative postal address",
+            classes: "govuk-fieldset__legend--m"
+          },
+          items: alternateAddressOptions
+        }) }}
+      {% endset -%}
+
+      {% set replyAddressHtml %}
+        {{ govukSelect({
+          id: "alternate-reply-address",
+          name: "alternateReplyAddress",
+          label: {
+            text: "Please pick an alternative reply address",
+            classes: "govuk-fieldset__legend--m"
+          },
+          items: replyAddressOptions
+        }) }}
+      {% endset -%}
+
+      {{ mojSideNavigation({
+        label: 'Side navigation',
+        items: [{
+          text: 'Basic details',
+          href: '/basic-details/'+breachNotice.id,
+          active: true
+        }, {
+          text: 'Warning type',
+          href: '/warning-type/'+breachNotice.id
+        }, {
+          text: 'Warning details',
+          href: '/warning-details/'+breachNotice.id
+        }, {
+          text: 'Next appointment',
+          href: '/next-appointment/'+breachNotice.id
+        }, {
+          text: 'Check your answers',
+          href: '/check-your-report/'+breachNotice.id
+        }]
+      }) }}
+
+    </div>
+
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-x1">Breach Notice - Basic Details</h1>
+      <b class="govuk-fieldset__legend--m">Title and Full Name
+      </b><br/>
+      <span id="title-and-full-name">{{ breachNotice.titleAndFullName }}</span>
+      <p/><br>
```

Comment:
`<p/>` is a self-closing paragraph tag, which I'm guessing isn't intentional.  Would be good to use `<p></p>` paragraphs instead of `<br/>`s - but not super important.

```suggestion
      <p>
        <div class="govuk-!-font-weight-bold">Title and Full Name</div>
        <span id="title-and-full-name">{{ breachNotice.titleAndFullName }}</span>
      </p>
```